controller/server/main.go

```diff
@@ -356,6 +357,60 @@ func (s *server) ShowTopology(ctx context.Context, req *cpb.ShowTopologyRequest)
 	return resp, nil
 }
 
+func (s *server) PushConfig(ctx context.Context, req *cpb.PushConfigRequest) (*cpb.PushConfigResponse, error) {
```

Comment:
why no tests?

Reply:
right now there is a test for this binary that only tests that the incoming request can be parsed into the correct struct for a deployment

there should be coverage for all of the grpc method implementations (not only the two new ones added), most of them are basically just calling the topology manager library